src/Temporalio/Common/Plugin.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Temporalio.Client;
+using Temporalio.Client.Interceptors;
+using Temporalio.Worker;
+using Temporalio.Worker.Interceptors;
+
+namespace Temporalio.Common
+{
+    public class Plugin : IClientPlugin, IWorkerPlugin
+    {
+        private IWorkerPlugin? nextWorkerPlugin;
+        private IClientPlugin? nextClientPlugin;
```

Comment:
Would recommend these be protected properties, and that they return non-null values and fail if the backing fields are null